projects/testing-library/src/lib/models.ts

```diff
@@ -282,6 +287,27 @@ export interface RenderDirectiveOptions<WrapperType, Properties extends object =
   componentProperties?: Partial<WrapperType & Properties>;
 }
 
+// eslint-disable-next-line @typescript-eslint/ban-types
+export interface RenderTemplateOptions<WrapperType, Properties extends object = {}, Q extends Queries = typeof queries>
+  extends RenderComponentOptions<Properties, Q> {
+  /**
+   * @description
+   * An Angular component to wrap the component in.
+   * The template will be overridden with the `template` option.
+   *
+   * @default
+   * `WrapperComponent`, an empty component that strips the `ng-version` attribute
+   *
+   * @example
+   * const component = await render(SpoilerDirective, {
+   *  template: `<div spoiler message='SPOILER'></div>`
+   *  wrapper: CustomWrapperComponent
+   * })
+   */
+  wrapper?: Type<WrapperType>;
+  componentProperties?: Partial<WrapperType & Properties>;
```

Comment:
Do you have an opinion about the naming of `componentProperties` ?
Should it be better to name this `wrapperComponentProperties` or `hostComponentProperties`?

Reply:
I like `wrapperComponentProperties` which has consistency with `wrapper`.

Reply:
But I think it is hard to change the name now. Because `RenderComponentOptions` and `RenderDirectiveOptions` still has `componentProperties`, making overload with co-existing these are complex.

Reply:
I'm thinking of two options.

Option 1. Just `properties` or `props` 

Vue or other families are adopting that. And it is not relevant directly, Storybook for Angular has also a similar API. This option brings breaking change for the entire package but the result is simple.

https://storybook.js.org/docs/angular/writing-stories/introduction

```ts
export default {
  title: 'Components/Button',
  component: Button,
} as Meta;

export const Primary = () => ({
  props: {
    label: 'Button',
    background: '#ff0',
  },
});
```


Option 2. Extending `wrapper` for passing wrapper component properties.  Implementation for this is more complex than Option 1.

```
{
  wrapper: {
    component: WrapperComponent,
    properties: { ... }
  }
}
```

Reply:
I like option 1, and you're (again) totally right.
Let's leave it with componentProperties, and we might rename that in a next breaking version.
I won't do it know, just because this is a minor difference.